python/ray/tune/examples/hyperband_example.py

```diff
@@ -58,13 +57,14 @@ def load_checkpoint(self, checkpoint_path):
         mode="max",
         max_t=200)
 
-    run(MyTrainableClass,
+    tune.run(
+        MyTrainableClass,
         name="hyperband_test",
         num_samples=20,
         stop={"training_iteration": 1 if args.smoke_test else 99999},
         config={
-            "width": sample_from(lambda spec: 10 + int(90 * random.random())),
-            "height": sample_from(lambda spec: int(100 * random.random()))
+            "width": tune.randint(10, 90),
```

Comment:
nit: this would be `randint(10, 100)`

Should we also remove mode/metric from the scheduler and pass it to tune.run here?

Reply:
will do this in followup!